drivers/staging/media/rpivid/rpivid_h265.c

```diff
@@ -20,7 +20,9 @@
 #include "rpivid_hw.h"
 
 #define DEBUG_TRACE_P1_CMD 0
-#define DEBUG_TRACE_EXECUTION 0
+#define DEBUG_TRACE_EXECUTION 1
```

Comment:
Did we mean to turn on tracing by default?

Reply:
No - fixed